chainercv/evaluations/eval_semantic_segmentation_iou.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import division
+
+import numpy as np
+import six
+
+
+def calc_semantic_segmentation_confusion(pred_labels, gt_labels):
+    """Collect a confusion matrix.
+
+    The number of classes :math:`n\_class` is computed as the maximum
+    class id among :obj:`pred_labels` and :obj:`gt_labels`.
```

Comment:
It sounds `n_class = max(pred_labels, gt_labels)`. `as` -> `from`? Perhaps, an equation like `n_class = max(pred_labels, gt_labels) + 1` will be helpful.